### PR TITLE
add buble sort

### LINKED ISSUE #11: implement bubble sort

### PR DESCRIPTION
**Pull request by QuinnMarshal**
closes #11 
add bubble sort algorithm

### CODE REVIEW

File: bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        for j in range(0, len(arr) - i - 1):
+            if(arr[j]>arr[j+1]):
+                temp = arr[j]
+                arr[j] = arr[j+1]
+                arr[j+1] = temp
+                swapped = True
+                
+        if not swapped:
+            break
+
```
Comment (keyvankhademi) [inline bubble_sort.py]: There are too many extra lines at the end.

**Review by keyvankhademi — DISMISSED** [review]

File: bubble_sort.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr):
+    for i in range(len(arr)):
+        swapped = False
```
Comment (EliasPinno) [inline bubble_sort.py:3]: I like this variable usage to ensure algorithm ends earlier. We should add some test cases later to ensure this does not cause issues though.

**Review by EliasPinno — DISMISSED** [review]

File: bubble_sort.py (hunk 1 above)
Comment (QuinnMarshal) [reply bubble_sort.py]: Okay I'm on it!

File: bubble_sort.py (hunk 2 above)
Comment (QuinnMarshal) [reply bubble_sort.py:3]: Thanks for sharing your thoughts! I agree

**Review by IvanIsCoding — APPROVED** [review]
LGTM

**Pull request merged**